rebasehelper/specfile.py

```diff
@@ -331,7 +331,7 @@ def get_not_used_patches(self):
         """
         return self.patches['not_applied']
 
-    def _comment_out_patches(self, patch_num, note=None):
+    def _comment_out_patches(self, patch_num, note=None, remove_patches=False):
```

Comment:
What do you exactly mean by comment_out list variable? I guess the plan is to either delete(deleted) or leave untouched(modified/applied).